pyproject.toml

```diff
@@ -38,7 +38,8 @@ dependencies = [
 	"toml==0.10.0",
 	"typing-inspect==0.8.0",
 	"XlsxWriter~=1.3.6",
-	"semver==3.0.0-dev.4"
+	"semver==3.0.0-dev.4",
+	"pandas~=2.0.1"
```

Comment:
I think we can achieve this without pandas, to limit the scale of dependencies

Reply:
removed pandas in favor of tabulate